mirage/config.js

```diff
@@ -85,6 +85,21 @@ export default function() {
 
   this.delete('/user_categories/:id');
 
+  this.get('/skills');
+  this.get('/user_skills');
+
+  // TODO: Make this work when relationships work
```

Comment:
You might even be able to outright remove the custom function and just have `this.post('/user_skills')`